integration-tests/testutils/autopkgtest/autopkgtest.go

```diff
@@ -80,7 +80,7 @@ func (a *AutoPkgTest) adtRun(testbedOptions string) (err error) {
 	prepareTargetDir(outputDir)
 
 	cmd := []string{
-		"adt-run", "-B",
+		"adt-run", "-B", "-q",
```

Comment:
Should this `-q` be conditional on the verbose setting in the config? I.e. not setting it when the config is set to verbose? Or is it generally just noise that is added is added when we not set the `-q`.